neo/Network/RPC/Models/RpcTransaction.cs

```diff
@@ -38,7 +41,7 @@ public static RpcTransaction FromJson(JObject json)
                 transaction.BlockHash = UInt256.Parse(json["blockhash"].AsString());
                 transaction.Confirmations = (int)json["confirmations"].AsNumber();
                 transaction.BlockTime = (uint)json["blocktime"].AsNumber();
-                transaction.VMState = json["vmState"].TryGetEnum<VMState>();
+                transaction.VMState = json["vmState"]?.TryGetEnum<VMState>();
```

Comment:
If it is `null` what value it take?

Reply:
Just null, this field uses a nullable type.